### PR TITLE
added sql reference

### PR DESCRIPTION
**Pull request by lwandzura**
added sql reference to the _index file

### CODE REVIEW

File: content/influxdb/cloud-iox/reference/sql/_index.md [hunk 1]
```diff
@@ -9,4 +9,597 @@ menu:
...
+
+SELECT 
+SELECTOR_LAST("water_level", time)['value'],
+SELECTOR_LAST("water_level", time)['time']
+FROM "h2o_feet"
+WHERE time >= timestamp '2019-09-10T00:00:00Z' AND time <= timestamp '2019-09-19T00:00:00Z'
+
+```
+
+### Time series functions
```
Comment (sanderson) [inline content/influxdb/cloud-iox/reference/sql/_index.md:537]: These are necessarily unique to "time series."

```suggestion
### Date and time functions
```

File: content/influxdb/cloud-iox/reference/sql/_index.md [hunk 2]
```diff
@@ -9,4 +9,597 @@ menu:
...
+
+| Function     | Description                                                                                     |
+| :----------- | :---------------------------------------------------------------------------------------------- |
+| DATE_BIN()   | Bins the input timestamp into a specified interval.                                             |
+| DATE_TRUNC() | Truncates a timestamp expression based on the date part specified, such as hour, day, or month. |
+| DATE_PART()  | Returns the specified part of a date.                                                           |
+| NOW()        | Returns the current time.                                                                       |
```
Comment (sanderson) [inline content/influxdb/cloud-iox/reference/sql/_index.md:544]:
```suggestion
| NOW()        | Returns the current time (UTC).                                                                  |
```

File: content/influxdb/cloud-iox/reference/sql/_index.md [hunk 3]
```diff
@@ -9,4 +9,597 @@ menu:
...
+SUM(water_level)
+FROM "h2o_feet"
+GROUP BY time
+```
+
+### Approximate functions
+
+| Function      | Description                                    |
+| :------------ | :--------------------------------------------- |
+| APPROX_MEDIAN | Returns the approximate median of input values. |
```
Comment (sanderson) [inline content/influxdb/cloud-iox/reference/sql/_index.md:564]: This is missing functions.

- approx_distinct
- approx_percentile_cont
- approx_percentile_cont_with_weight

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud-iox/reference/sql/_index.md (hunk 3 above)
Comment (lwandzura) [reply content/influxdb/cloud-iox/reference/sql/_index.md:564]: adding now

File: content/influxdb/cloud-iox/reference/sql/_index.md [hunk 4]
```diff
@@ -9,4 +9,600 @@ menu:
...
+| :------: | :------------------------------------ | :--------------- |
+|   `=`    | Equal to                              | `123 = 123`      |
+|   `<>`   | Not equal to                          | `123 <> 456`     |
+|   `!=`   | Not equal to                          | `123 != 456`     |
+|   `>`    | Greater than                          | `3 > 2`          |
+|   `>=`   | Greater than or equal to              | `3 >= 2`         |
+|   `<`    | Less than                             | `1 < 2`          |
+|   `<=`   | Less than or equal to                 | `1 <= 2`         |
+|   `~`    | Matches a regular expression          | `'abc' ~ 'a.*'`  |
+|   `!~`   | Does not matches a regular expression | `'abc' !~ 'd.*'` |
```
Comment (sanderson) [inline content/influxdb/cloud-iox/reference/sql/_index.md:152]:
```suggestion
| Operator | Meaning                             | Example          |
| :------: | :---------------------------------- | :--------------- |
|   `=`    | Equal to                            | `123 = 123`      |
|   `<>`   | Not equal to                        | `123 <> 456`     |
|   `!=`   | Not equal to                        | `123 != 456`     |
|   `>`    | Greater than                        | `3 > 2`          |
|   `>=`   | Greater than or equal to            | `3 >= 2`         |
|   `<`    | Less than                           | `1 < 2`          |
|   `<=`   | Less than or equal to               | `1 <= 2`         |
|   `~`    | Matches a regular expression        | `'abc' ~ 'a.*'`  |
|   `!~`   | Does not match a regular expression | `'abc' !~ 'd.*'` |
```

File: content/influxdb/cloud-iox/reference/sql/_index.md [hunk 5]
```diff
@@ -9,4 +9,600 @@ menu:
...
+
+## Keywords
+
+The following reserved key words cannot be used as identifiers.
```
Comment (sanderson) [inline content/influxdb/cloud-iox/reference/sql/_index.md:156]:
```suggestion
The following reserved keywords cannot be used as identifiers.
```

**Review by sanderson — APPROVED** [review]

**Pull request merged**